starters/next-wordpress-starter/next.config.js

```diff
@@ -38,6 +38,17 @@ if (process.env.WPGRAPHQL_URL === undefined) {
 // remove trailing slash if it exists
 imageDomain = imageDomain.replace(/\/$/, '');
 
+let PANTHEON_ENVIRONMENT, IS_LIVE_ENVIRONMENT;
```

Comment:
I would expect these to either be added to `nextConfig.env` or `process.env` for use throughout the application as needed.